### PR TITLE
SDK 2.0: Clean Out old SDK types

### PR DESCRIPTION
**Pull request by xitman78**
Clean out of old SDK.
Documentation updated with new SDK types.
Removed unused functions in `api.ts`

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-348.gearboxjs.preview.cogniteapp.com

File: src/components/SensorOverlay/stories/full.md [hunk 1]
```diff
@@ -100,9 +100,18 @@ interface SensorMinMaxRange {
 }
 ```
 
-#### Datapoint
-This interface is provided by [@cognite/sdk](https://github.com/cognitedata/cognitesdk-js):
+#### SensorDatapoint
+This type can be imported from `@cognite\gearbox`:
```
Comment (mrtraser) [inline src/components/SensorOverlay/stories/full.md]: revert slash sign

File: src/components/SensorOverlay/stories/full.md [hunk 2]
```diff
@@ -100,9 +100,18 @@ interface SensorMinMaxRange {
...
 ```typescript
-import { Datapoint } from '@cognite/sdk';
+import { SensorDatapoint } from '@cognite\gearbox';
```
Comment (mrtraser) [inline src/components/SensorOverlay/stories/full.md]: slash

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-348.gearboxjs.preview.cogniteapp.com

File: src/components/SensorOverlay/stories/full.md (hunk 2 above)
Comment (xitman78) [reply src/components/SensorOverlay/stories/full.md]: fixed

File: src/components/SensorOverlay/stories/full.md (hunk 1 above)
Comment (xitman78) [reply src/components/SensorOverlay/stories/full.md]: fixed

**Pull request merged**